### DISCUSSION TITLE
When does Vercel Network support IPv6 and HTTP3

### DISCUSSION THREAD

**Discussion by Fgaoxing** [vercel/community#5848] [Feature Question] [answered]
### Question

I know I asked this question again, but there was no further response, so I hope to clarify it again. And do not close this "discussions" until it is supported

**Comment by Fgaoxing** [comment]
@rauchg

**Comment by swarnava** [comment, marked as answer]
Hi,

We are working closely with upstream provider to bring support for IPv6 address.  We don't currently support HTTP3, however this is on our roadmap of items we want to add in the future but we don't have an ETA or able to set any expectation of when this will be added.

Thanks!
